entities/component.py

```diff
@@ -212,11 +212,8 @@ def add_variant(self, variant: Variant) -> None:
         self.variants.append(variant)
 
     def serialize(self, output_directory: str) -> None:
-        dir_path = path.join(output_directory, self.uuid)
-        if not (path.exists(dir_path) and path.isdir(dir_path)):
-            makedirs(dir_path)
-        with open(path.join(dir_path, '.librepcb-cmp'), 'w') as f:
-            f.write('1\n')
-        with open(path.join(dir_path, 'component.lp'), 'w') as f:
-            f.write(str(self))
-            f.write('\n')
+        serialize_common(serializable=self,
+                         output_directory=output_directory,
+                         uuid=self.uuid,
+                         long_type='component',
+                         short_type='cmp')
```

Comment:
Nit: Could you format the function calls like this instead?

```
serialize_common(
    serializable=self,
    output_directory=output_directory,
    uuid=self.uuid,
    long_type='component',
    short_type='cmp',
)
```

The reason is that if we ever rename the `serialize_common` function, we don't need to change every line in the function call just to re-align the arguments vertically...

Reply:
See https://github.com/LibrePCB/librepcb-parts-generator/pull/133 for a suggestion to use ruff for formatting.